docs/proposals/http-triggers.md

```diff
@@ -0,0 +1,50 @@
+# http trigger improvements
+
+Though there is no standard on what http/https triggers of FAAS platform should support, most hosted FAAS solution like AWS Lambda, google cloud functions, IBM cloud functions, Azure functions etc provide common functionality
+
+- http/https endpoint for the function: a fully qualified URL is automatically generated and assigned to an HTTP triggered Cloud Function which can retrieved through respective cli or consoles
+- ability to customize the endpoint url by specifying route
+- a way to authenticate and authorize the function invoker through url
+- a way to restrict http method GET/POST/DELETE etc used to invoke function
+- stage and version functions
```

Comment:
All these points are true but you should specify which issues you want to tackle with this specific proposal

Reply:
Is it not clear with last two sections? Any way I will add specific issue numbers to the proposal.